CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Change history for ui-servicepoints
 
+## [2.1.0] (IN PROGRESS)
+
+*Error accessing Requests when user doesn't have a service point selected.  Fixes UIREQ-440.
```

Comment:
@K-Felk would you mind adding a space after `*`? Thank you!

Reply:
Done and change wrapped into merge request.